src/DotNetWorker/Configuration/WorkerOptions.cs

```diff
@@ -1,29 +1,27 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core.Serialization;
 
-namespace Microsoft.Azure.Functions.Worker.Configuration
+namespace Microsoft.Azure.Functions.Worker
 {
-    // TODO: Slim this down.
     public class WorkerOptions
     {
         public WorkerOptions()
         {
-            Capabilities = new List<string>();
+            var serializerOptions = new JsonSerializerOptions
+            {
+                PropertyNameCaseInsensitive = true
+            };
 
-            // Default value for HeartBeatRateFactor is 6
-            HeartBeatRateFactor = 6;
+            Serializer = new JsonObjectSerializer(serializerOptions);
```

Comment:
Should this be deferred? If a different serializer is used, we shouldn't need to create this instance, right?